.ci-config/rippled.cfg

```diff
@@ -170,3 +170,5 @@ fixNFTokenRemint
 # 2.0.0-b4 Amendments
 XChainBridge
 DID
+# 2.2.0-b1 Amendments
+PriceOracle
```

Comment:
The fix amendments should probably all be added here too

Reply:
updated in [df462ba](https://github.com/XRPLF/xrpl-py/pull/701/commits/df462ba5ca42b12e4ab754c9c6446b990264f6da)